docs/proposals/koordlet/20230728-support-cold-memory-compute.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Support to collect and report cold memory
+authors:
+  - "@BUPT-wxq"
+reviewers:
+  - "@saintube"
+  - "@zwzhang0107"
+  - "@eahydra"
+  - "@hormes"
+  - "@FillZpp"
+creation-date: 2023-07-28
+last-updated: 2023-07-28
+---
+
+## Table of Contents
+
+
+- [Title](#title)
+  - [Table of Contents](#table-of-contents)
+  - [Glossary](#glossary)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals/Future Work](#non-goalsfuture-work)
+  - [Proposal](#proposal)
+    - [Design](#design)
+    - [Collect idle memory info and insert memory with hot page](#collect-idle-memory-info-and-insert-memory-with-hot-page)
+	- [Report memory usage including hot page](#report-memory-usage-including-hot-page)
+	- [Define cold memory API](#define-cold-memory-api)
+  - [Alternatives](#alternatives)
+
+## Glossary
+
+**kidled**
+
+Kided, an open source cold memory collection solution, identifies the hot and cold conditions of nodes, pod and container memory in the cluster. The link introduces kildled in detail.
```

Comment:
Since the kidled is in Anolis OS instead of the upstream Linux, We also need a section to demonstrate how this work gets compatible or degraded to the Linux kernels without kidled.